python/grass/jupyter/utils.py

```diff
@@ -202,6 +202,21 @@ def get_rendering_size(region, width, height, default_width=600, default_height=
     return (default_width, round(default_width * region_height / region_width))
 
 
+def get_computational_region_bb():
+    """
+    Gets the current computational region.
+    """
+    region = gs.parse_command("g.region", flags="bg")
+    return [(region["ll_s"], region["ll_w"]), (region["ll_n"], region["ll_e"])]
+
+
+def update_region(north, south, east, west):
+    """
+    Updates the GRASS GIS region using the given coordinates.
+    """
+    gs.run_command("g.region", n=north, s=south, e=east, w=west)
```

Comment:
This should behave the same as in the gui:
https://github.com/OSGeo/grass/blob/main/gui/wxpython/mapwin/buffered.py#L1633

so you need to additionally specify the resolution to keep it the same

Reply:
This is the output given by `region = gs.parse_command("g.region", flags="bg")`:
`{'ll_n': '35.82917819',
 'll_s': '35.70582758',
 'll_w': '-78.61589871',
 'll_e': '-78.44816944',
 'll_clon': '-78.53203408',
 'll_clat': '35.76750289'}
 `
I am not able to understand why the resolution key is not here. However, if I run it without the flag "b", it gives me the resolution keys.

Reply:
`gs.parse_command("g.region", flags='gbp')` will give you the resolution also.